addons/website_livechat/static/src/web/website_livechat_notification_handler.js

```diff
@@ -7,8 +7,7 @@ export const websiteLivechatNotifications = {
             const { Thread } = store.insert(payload);
             for (const thread of Thread) {
                 const chatWindow = store.ChatWindow.insert({ thread });
-                chatWindow.open();
-                chatWindow.focus();
+                chatWindow.open({ focus: true });
```

Comment:
I'd like to remove this whole notif handler.
This might be replaceable by having `open_chat_window` in the payload (and just use record insert rather than "website_livechat.send_chat_request").

Reply:
Done!